src/ext/zrapp/availexps.ml

```diff
@@ -243,11 +243,14 @@ let eh_handle_inst i eh =
       (eh_kill_mem eh;
        eh_kill_addrof_or_global eh;
        eh)
+  (* todo: didn't find location where stmts are handled *)
```

Comment:
We'll have to look up `zrapp` again to decide what to do there.

Reply:
- The main.ml changes were unintentionally forgotten testing utilities.
- asmlabelslst_ne is not used in first step because we semantically distinguish between:
  - labels field was provided (Some [...]) 
  - labels field was omitted (None)
- the check asm function ensures that goto attribute <=> labels field (otherwise we would need state in the parser).
  it doesn't return unit because for whatever reason when I write in cparser.mly inside asm rule:
  `{ let asm = ASM ($1 ....) in checkAsm asm; asm }` dune build fails with a syntax error in cparser.ml (can't really debug that because it's generated.
- as cil cfg is only used to make the goblint cfg, we don't do anything special for the cil one
- `ASM a` does not work for some reason (replying to the comment in email)
- I don't know what to do for the dummy instruction (do we need to introduce a new instruction type for that?)